src/components/preferences/Webhooks.js

```diff
@@ -82,11 +90,10 @@ export default function Webhooks(props) {
                         name="webhook.type.type"
                         id={buildID(baseId, ids.WEBHOOK_TYPES_SELECT)}
                         label={t("type")}
-                        required
                         select
-                        variant="outlined"
+                        variant="standard"
                         size="small"
-                        component={FormTextField}
+                        component={FormSelectField}
```

Comment:
If I add field level validation, it will work but I have to touch the field first 😕 . Also I had to change it `FormSelectField` instead of `FormTextField`

Reply:
Having to touch the field first sounds right to me. If the user opens a blank form, for example, we don't want every field to automatically be red just because they haven't filled it out yet. So validation should run after a field has been touched or when the user tries to submit.